templates/case-study/nasa.html

```diff
@@ -0,0 +1,54 @@
+{% extends "case-study/base_case-study.html" %}
```

Comment:
are we planning to remove this and the other (example.html) case studies before we merge to production?

Reply:
Yes, before the feature branch is removed the example case studies will be removed. They are currently included for QA purposes only.